templates/vue/src/components/Lightbox/media/WpPostMedia.vue

```diff
@@ -71,15 +72,18 @@ export default {
   }
 
   article {
-    color: #47425d;
+    color: #111;
```

Comment:
Again, not sure if we need this on a child element - can we keep it on the parent elements instead? Is there an overwrite somewhere that should be removed instead?

Reply:
Yep, moved to parent (per previous...), moving to parent doesnt seem to break anything on my end